cpanfile

```diff
@@ -37,6 +37,7 @@ requires 'PGObject::Type::DateTime', '1.0.4';
 requires 'PGObject::Type::ByteString', '1.1.1';
 requires 'PGObject::Util::DBMethod';
 requires 'PGObject::Util::DBAdmin', '0.09';
+requires 'Plack';
```

Comment:
Pulling Plack as a whole brings many things, Request & Response, for example, that we felt unneccessary before. Should we be more precise?